### ISSUE TITLE
Cannot enter Undercity. None of the Undercity elevator doors open. 

### ISSUE THREAD

**Issue by pj4041** [ValorenWoW/sunstrider-core#5465]
**Describe the bug**
None of the Undercity elevator doors open.

**To Reproduce**
1. Go to Undercity , Lordaeron (ground) level
2. Attempt to enter the actual Undercity (underground) level
3. Doors never open for any of the elevators.

**Expected behavior**
Doors must open every couple of seconds.

**Comment by pj4041** [comment]
Update:
I've had another player come with me to confirm this. 
There is a teleporter NPC nearby so we used it to teleport to Shattrath, then entered Undercity portal to appear inside the city. We were able to use the elevators to climb up and exit the city as expected. From then on the doors appear to function as expected, both ways.

I am not sure what happened but I am sure the doors did not open before. They were actually visually glitching every few seconds.
